app/javascript/react/src/boozeYourself.js

```diff
@@ -10,7 +12,13 @@ class BoozeYourself extends Component {
 
   render(){
     return(
-      <h1>Hello from the react front end</h1>
+
+      <Router>
+
+        <Route path='/' component={NavBar} />
+
+      </Router>
+
     )
   }
 
```

Comment:
A lot of extra spacing in this file.  I'd recommend bringing it in a bit delete the spacing on lines (24, 21, 19, 17, and 9).